rj_gameplay/classifier/pass_success_classifier.py

```diff
@@ -0,0 +1,95 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+
+import numpy as np
+import math as m
+
+def classifier(world_state: rc.WorldState) -> bool:
```

Comment:
Nitpick, but just call this function pass_success_classifier or something more descriptive